server/env.js

```diff
@@ -0,0 +1,32 @@
+// WARNING! All process.env values are strings, so check for 'true' or 'false'
+
+const {
+  NODE_ENV,
+  // OPEN_ALL_CONTESTS,
+  USE_FIREBASE,
+  ENV_LEVEL,
+  LOG_LEVEL: LOG_LEVEL_DOTENV,
+  ENV_PORT,
+} = process.env;
```

Comment:
Nit, could you sort this list (and the exports) alphabetically? It makes it easier to maintain and edit in the future.

Reply:
Done